Added elasticsearch db, other minor changes

masroorhasan opened the pull request:
- Integrated ES
- Ran basic tests for ES Adapter
- Minor modifications

theProf commented on web/api/controllers/CVController.js:
```diff
@@ -66,8 +66,8 @@ module.exports = {
-      for (var i = 1; i < 3; i++){
-          for (var j = 1; j < 7; j++){
+      for (var i = 1; i < 7; i++){
+          for (var j = 1; j < 4; j++){
```
at least use variable names for hard-coded values

theProf commented on web/api/controllers/CVController.js:
```diff
@@ -80,7 +80,7 @@ module.exports = {
       
       var predictedImg = {};
       // cv.readImage("/Users/masroorhasan/Downloads/att_faces/s6/9.pgm", function(e, im){
-      cv.readImage(path.resolve(__dirname, '../../assets/images/out/warren11.pgm'), function(e, im){    
```
variable names for hard-coded values

theProf commented on web/api/controllers/ImageController.js:
```diff
@@ -87,9 +101,18 @@ var ImageController = {
+
+                                    // var out_png = path.resolve(__dirname, '../../assets/images/out/');
+                                    // out_png += "/" + model.id + ".png";
+                                    // im.save(out_png);
```
remove comments and any unused code

theProf commented on web/package.json:
```diff
@@ -5,23 +5,25 @@
+    "sails": "~0.10.0-rc7",
+    "sails-disk": "~0.10.0",
+    "waterline-elasticsearch": "^0.1.3"
   },
```
include all module:version additions/changes in commit description

theProf commented on web/package.json:
```diff
@@ -34,4 +36,4 @@
   "license": ""
-}
\ No newline at end of file
+}
```
whitespace changes in their own commit

theProf committed:
Added elasticsearch db, other minor changes

Node Module Changes:
** Removed **
  - grunt-contrib-coffee    : ~0.10.1
** Added **
  - grunt-contrib-uglify    : ~0.4.0
  - grunt-contrib-watch     : ~0.5.3
  - grunt-sails-linker      : ~0.9.5
  - grunt-sync              : ~0.0.4
  - include-all             : ~0.1.3
  - rc                      : ~0.3.3
  - sails                   : ~0.10.0-rc7
  - sails-disk              : ~0.10.0
  - waterline-elasticsearch : ^0.1.3

theProf committed:
whitespace changes

theProf commented:
looks good
